fili-core/src/main/java/com/yahoo/bard/webservice/web/util/PaginationParameters.java

```diff
@@ -90,6 +93,11 @@ private int parseParameter (String parameter, String parameterName) throws BadPa
             throw new BadPaginationException(errorMessage.format(parameterName));
         }
         try {
+            if (parameter.equals(FIRST)) {
+                return 1;
+            } else if (parameter.equals(LAST)) {
+                return LAST_PAGE;
```

Comment:
I wasn't sure what to do about this, so I just left it for now. I would be happy to make any changes.